README.md

```diff
@@ -1,19 +1,311 @@
+      "licenceType": "ScrapMetalMobileCollector",
+      "verifier": {
+         "crn": "1234567890"
+      }
+   },
+   "dateTimeChecked": "2021-12-02T15:25:57.728Z[Europe/London]",
+   "status": "NoMatch"
+}
+```
+
+### `GET /hec/unexpired-tax-checks`
+Requires an active GG session. Gets summary data of all tax checks associated with the GG account associated 
+with the GG session. Returns `200` (OK) if successful. If not tax checks are found, a `200` is still returned -
+the response will just contain an empty array.
+
+Example response body:
+```json
+[
+   {
+      "licenceType": "ScrapMetalDealerSite", 
+      "taxCheckCode": "PPL99KT6P",
+      "expiresAfter": "2022-04-01",
+      "createDate":"2021-12-02T15:35:20.638Z[Europe/London]"
+   },
+   {
+      "licenceType": "ScrapMetalDealerSite",
+      "taxCheckCode": "GALH79T79",
+      "expiresAfter": "2022-04-01",
+      "createDate":"2021-12-02T11:53:02.004Z[Europe/London]"
+   }
+]
+```
+
+### `GET /hec/sa-status/:utr/:taxYear`       
+Get the self assessment (SA) return status for the given SA UTR and the tax year. The tax year
+is the start year of the tax year, e.g. for the 2021/2022 tax year the input should be 2021. A 
+`200` (OK) repsonse is given if the return status can be returned. Requires an active GG session.
+
+Example response body:
+```json
+{
+   "sautr": "1234567890",
+   "taxYear": 2020,
+   "status": "NoReturnFound"
+}
+```
+
+### `GET /hec/ct-status/:utr/:startDate/:endDate`
+Attempt to find the latest corporation tax (CT) accounting period within the given lookup period and the 
+return status of that accounting period if one exists. A`200` (OK) repsonse is given if the return status can be 
+returned. Requires an active GG session.
+
+Example response body:
+```json
+{
+   "ctStatus": {
+      "ctutr":  "1234567890",
+      "startDate":  "20201001",
+      "endDate":  "20193009",
+      "latestAccountingPeriod": {
+         "startDate": "20200131",
+         "endDate": "20200524",
+         "ctStatus": "NoReturnFound"
+      }
+   }
+}
+```
 
+### `GET /hec/ctutr/:crn`                          
+Return the CT UTR associated with the given CRN (company registration number). Returns a `200` (OK) response
+if a CT UTR can be found. If no CT UTR can be found a `404` (NOT FOUND) response is given. Requires an
+active GG session.
+
+Example response body:
+```json
+{
+   "ctutr": "1234567890"
+}
+```
+
+### `POST /hec/file-transfer/callback`
+Handles a callback which is triggered from this service notifying downstream systems that a file is ready to 
+pick up (c.f. [HEC Tax Check Data File Transfer](#hec-tax-check-data-file-transfer)). A `200` (OK)
+response with no body will be returned if the callback has been handled successfully.
+
+Example request body:
+```json
+{
+   "notification": "FileProcessingFailure",
+   "filename": "file.txt",
+   "correlationID": "12345",
+   "failureReason": "¯\\_(ツ)_//¯"
+}
+```
+
+### `POST       /hec/test-only/tax-check`              
```

Comment:
May be put a line or any type of separation to show test-only end points, pls ignore if not relevant

Reply:
updated [here](https://github.com/hmrc/hec/pull/53/commits/029cc5817c153a21799cc847141b6f5f18e2e6c3)